src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -0,0 +1,44 @@
+package snyk.amplitude
+
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.diagnostic.logger
+import snyk.amplitude.api.AmplitudeExperimentApiClient
+import snyk.amplitude.api.AmplitudeExperimentApiClient.Defaults.FALLBACK_VARIANT
+import snyk.amplitude.api.ExperimentUser
+import snyk.amplitude.api.Variant
+import java.util.concurrent.ConcurrentHashMap
+
+@Service
+class AmplitudeExperimentService : Disposable {
+    private val log = logger<AmplitudeExperimentService>()
+
+    private val apiClient: AmplitudeExperimentApiClient =
+        AmplitudeExperimentApiClient.create(apiKey = "client-bQAwhmzLhgpgqAhKAqRzIjnS915YrAqj")
```

Comment:
hardcoded key?

Reply:
The envkey is hardcoded to simplify development, maybe later we'll make it configurable. Compare https://github.com/snyk/vscode-extension/blob/feat/experiment/src/snyk/experiments/amplitude.ts#L11